CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 cmake_minimum_required(VERSION 3.13)
```

Comment:
cmake output has cloned entries, doesn't seem entirely clean to me:
-- Boost version: 1.67.0
-- Boost version: 1.67.0
-- Found the following Boost libraries:
--   system
--   graph
-- Boost version: 1.67.0
-- Boost version: 1.67.0
-- Boost version: 1.67.0
-- Boost version: 1.67.0

Reply:
Did as  much as I could.